crates/cxx-qt-gen/src/generator/naming/signals.rs

```diff
@@ -51,6 +53,13 @@ impl CombinedIdent {
             rust: ident,
         }
     }
+
+    fn connect_from_signal(ident: &Ident) -> Self {
+        Self {
+            cpp: format_ident!("{}Connect", ident.to_string().to_case(Case::Camel)),
```

Comment:
Why not `on{}` with `Case::Pascal` here?
Would that conflict with QML?

If so, please add a comment about it here.

Reply:
I think that's what i was concerned about, that the C++ method then could start conflicting with the QML API and seem "odd". I'll add a comment if that was my concern...

Reply:
Done